server/impl/light-page-impl.js

```diff
@@ -7,7 +7,7 @@ function addLightPageHeaders(result, lightPages, { config, res, client, req }) {
     false
   );
 
-  if (typeof lightPages === "function" && !lightPages(config)) {
+  if (typeof lightPages === "function" && !lightPages(result.config || config)) {
```

Comment:
Why do we need this?

Reply:
result have `pagebuilder-config` we want to read amp-story page through pb.